code/game/verbs/ooc.dm

```diff
@@ -283,4 +283,50 @@ GLOBAL_VAR_INIT(admin_ooc_colour, "#b82e00")
 
 	init_verbs()
 
+/client/verb/show_own_notes()
+	set name = "Show Public Notes"
```

Comment:
`Show My Notes` maybe?

`Show Public Notes` sounds like you're viewing current server advisories.